source/solutions-library.txt

```diff
@@ -388,7 +388,7 @@ kick-start their projects.
 
          .. card::
             :headline: Streamline Global Gaming Management
-            :url: https://deploy-preview-202--docs-atlas-architecture.netlify.app/solutions-library/Streamline-global-gaming-management/
+            :url: /solutions-library/Streamline-global-gaming-management/
```

Comment:
(question)
Mostly we ignore casing in URLs. This is one of a couple examples where we introduce upper case lettering in the URL. Do we need to fix this up prior to publishing?